x/perpetual/spec/bot.md

```diff
@@ -14,10 +14,24 @@
 ### Requirements:
 
 1. **Stop-Loss Trigger:**
-   - Trigger the `closePosition` function if the `custody` price reaches the stop-loss price.
-   - **Required Data:**
-     - `custody` price (obtained using the AMM module).
-     - Stop-loss price (present in the `Position` object).
+  - Trigger the `closePosition` function if the `TradingAsset` price reaches the stop-loss price.
+  - **Required Data:**
+    - `TradingAsset` price (obtained using the AMM module).
```

Comment:
@amityadav0 only oracle pools are used by Perpetual, therefore the bot should always relying on the Oracle module for retrieving asset price.